WordPress/src/main/java/org/wordpress/android/networking/GlideRequestFactory.kt

```diff
@@ -41,16 +41,18 @@ class GlideRequestFactory @Inject constructor(
 
     private fun addAuthHeaders(url: String, currentHeaders: Map<String, String>): MutableMap<String, String> {
         val headers = currentHeaders.toMutableMap()
-        headers.put("User-Agent", userAgent.userAgent)
+        headers["User-Agent"] = userAgent.userAgent
         if (WPUrlUtils.safeToAddWordPressComAuthToken(url)) {
-            headers.put("Authorization", "Bearer " + accessToken.get())
+            accessToken.get()?.let {
```

Comment:
The access token is [by default the empty string](https://github.com/wordpress-mobile/WordPress-FluxC-Android/blob/9dc166d6c67d807d5ed6927a0fd65dd1cd979b82/fluxc/src/main/java/org/wordpress/android/fluxc/network/rest/wpcom/auth/AccessToken.java#L17), so this will still attach the invalid auth header. You can check if the token is valid by calling `accessToken.exists()` instead.